pt_br translation for guerrilla text

netrik182 opened the pull request.

lh1008 commented:
Thanks for the PR @netrik182, we are currently looking for a reviewer to continue with the process :).

netrik182 commented:
> 1- Change this part "apresente-os os benefícios do Monero." for "apresente a eles os benefícios do Monero.". It's better to read and don't have redundancy.
> 
> 2- Change "ferramentas **disponibilidades** pelo Monero Outreach" to "ferramentas **disponibilizadas** pelo Monero Outreach". Maybe it was a typo.

Thank you @leoassuncao. I've added your suggestion and fixed the typo.

lh1008 commented:
Are these changes approved @leoassuncao?
